Library/Homebrew/extend/ARGV.rb

```diff
@@ -141,6 +141,10 @@ def build_universal?
     include? '--universal'
   end
 
+  def build_dsym?
+    flag? '--dsym'
```

Comment:
You should use `include?` here because `flag?` will check for both `--dsym` *and* `-d`, which is already a short alias for `--debug`.